man/covidcast.Rd

```diff
@@ -47,6 +47,8 @@ Mutually exclusive with \code{as_of} or \code{issues}.}
 }
 \description{
 API docs: \url{https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html}
+}
+\details{
```

Comment:
Hm, you ran `devtools::document()` right? I would just trust whatever it outputs, as long as the @seealso format is right (which seems good to me).

Reply:
I ran `devtools::check()`, which calls `document`, right? It seemed plausible to me, just wanted to flag as a bit odd given I hadn't otherwise edited the area

Reply:
yea, it's a little weird. we'll need to take a look at the compiled docs after we merge.

and yea, `check` should document on its own.